transformers/component_tests/queries/all_urn_states_query_test.go

```diff
@@ -1,218 +0,0 @@
-package queries
-
-import (
-	"math/rand"
-	"strconv"
-
-	"github.com/makerdao/vdb-mcd-transformers/test_config"
-	helper "github.com/makerdao/vdb-mcd-transformers/transformers/component_tests/queries/test_helpers"
-	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
-	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/test_helpers"
-	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/vat"
-	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
-	"github.com/makerdao/vulcanizedb/libraries/shared/storage/types"
-	"github.com/makerdao/vulcanizedb/pkg/core"
-	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres/repositories"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-var _ = Describe("Urn history query", func() {
```

Comment:
And here it goes 💥 because we can just `select * from api.urn_snapshot where urn_identifier = x and ilk_identifier = y`?

Reply:
Exactly.